### ISSUE TITLE
Sway does not turn screen on after resume from suspend

### ISSUE THREAD

**Issue by desultory** [swaywm/sway#8172]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on Libera Chat.
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway.
- Please do NOT submit issues for information from the github wiki. The github wiki is community maintained and therefore may contain outdated information, scripts that don't work or obsolete workarounds.
  If you fix a script or find outdated information, don't hesitate to adjust the wiki page.

### Please fill out the following:
- **Sway Version:**
1.9

- **Configuration File:**
```
set $lock swaylock --show-failed-attempts \
	--color 1e1e1e --inside-color cccccc --ring-color ffffff \
	--inside-clear-color 11a8cd --ring-clear-color 29b8db \
	--inside-ver-color 2472c8 --ring-ver-color 3b8eea \
	--inside-wrong-color cd3131 --ring-wrong-color f14c4c

exec swayidle -w \
    timeout 10 'swaymsg "output * power off"' \
	resume 'swaymsg "output * power on"' \
    timeout 30 "loginctl suspend" \
	after-resume 'swaymsg "output * enable"' \
	before-sleep "$lock"
```

- **Description:**
  - The steps you took in plain English to reproduce the problem.
When the suspend resumes, the screen does not turn on until after i unlock swaylock.

https://gist.github.com/desultory/db6dc9221bf292bf4001c06c630155f2

**Comment by apirogov** [comment]
I also have this or a similar issue.

```
set $swaylock_cmd 'swaylock -f -c 000000'

exec swayidle -w \
         timeout 300 $swaylock_cmd \
         timeout 600 'swaymsg "output * power off"' \
         resume 'swaymsg "output * power on"' \
         before-sleep $swaylock_cmd

```

One of the external displays stays off with high likelihood, but reliably wakes up once I restart sway. The other, older external display has no issues whatsoever.

**Comment by sriedel** [comment]
To add to this: for me the screen non-enabling on wakeup seems to be non-deterministic, but happens more often than having the screen being enabled.
Sometimes, though rarely, reloading sway does enable the screen.
Unplugging the DP cable and re-inserting it does always immediately enable the screen though. Since I haven't had this behavior in the early days of sway, I'm guessing it's not really a hardware issue, but that sway doesn't enable the display reliably for some reason. Due to the non-deterministic nature, it _feels_ like a timing issue (e.g. sway probes for attached displays, but the external screen is too slow to respond after it itself was in deep-sleep mode, something like that).
